hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -153,19 +153,20 @@
       sw_test: sw/device/tests/dif_plic_sanitytest
     }
     {
-      name: chip_flash_ctrl_test
+      name: chip_flash_ctrl_access
       uvm_test_seq: chip_sw_base_vseq
       sw_test: sw/device/tests/flash_ctrl_test
     }
     {
-      name: chip_sha256_test
+      name: chip_hmac_sha256_encr
       uvm_test_seq: chip_sw_base_vseq
       sw_test: sw/device/tests/sha256_test
     }
     {
-      name: chip_rv_timer_test
+      name: chip_rv_timer_irq
       uvm_test_seq: chip_sw_base_vseq
-      sw_test: sw/device/tests/rv_timer_test
+      sw_test: sw/device/tests/dif_rv_timer_sanitytest
```

Comment:
How can we ensure that the changes made to the SW test like these do no break the DV (its hard to keep up with the PR reviews and we cannot run *everything* in the private CI)? 
@gkelly, any ideas?

Reply:
It might be useful to have a way of taking `chip_sim_cfg.hjson` and running each `sw_test` in verilator in the main CI? That should catch issues where the configuration is not updated, i think.

Reply:
These tests are already running in Verilator in CI. Its the DV setup that is getting stale because of the file renaming. We need to ensure that when existing tests get "ported" to DIF sanity, the corresponding entry in `chip_sim_cfg.hjson` is also updated.

Reply:
Maybe we can add a python test to cover this for now?

Reply:
@sriyerg the point would be to use the configuration file to choose which verilator tests get run in CI (rather than using the shell script), specifically so that the configuration stays up to date.

Reply:
I see. DVSim does not support Verilator batch runs yet but I absolutely think that would be very beneficial and useful.